mlir/test/Transforms/normalize-memrefs.mlir

```diff
@@ -0,0 +1,22 @@
+// RUN: mlir-opt -allow-unregistered-dialect -normalize-memrefs %s | FileCheck %s
+
+// CHECK-LABEL: func @matmul
```

Comment:
You can move these below - to right above the function.

Reply:
I didn't understand this.
Am I to bring all the 'CHECK*'s right above the function?
OR
Just the 'CHECK-LABEL' part should be right above the function?
(I have used 'CHECK-SAME' that logically works with 'CHECK-LABEL', so I presume that needs to be moved as well)

Reply:
I meant to just move these two CHECK lines to right above the function, which would be

```
// CHECK-LABEL: func @matmul
// CHECK-SAME: (%[[A:arg[0-9]+]]: memref<4x4xf64>, %[[B:arg[0-9]+]]: index, %[[C:arg[0-9]+]]: memref<8xf64>) -> memref<4x4xf64>
func @matmul(%A: memref<16xf64, #tile>, %B: index, %C: memref<8xf64>) -> (memref<16xf64, #tile>) {
```

Reply:
Oh okay, cool, thank you!